tox.ini

```
@@ -0,0 +1,44 @@
+[tox]
+envlist =
+    check
+    py37,
+    py38,
+    coverage
+    docs
+
+[testenv]
+usedevelop = True
+extras = test
+setenv =
+    PYTHONPATH = {toxinidir}
+    py{37,38}: COVERAGE_FILE=.coverage.{envname}
```

```suggestion
    COVERAGE_FILE=.coverage.{envname}
```
Turns out we don't need that here